.env

```diff
@@ -6,7 +6,7 @@ CA_CERTS_TRUSTED_STORE=/etc/ssl/certs
 # NeoGo privnet
 #CHAIN_PATH="/path/to/devenv.dump.gz"
 CHAIN_URL="https://github.com/nspcc-dev/neofs-contract/releases/download/v0.15.3/devenv_mainchain_notary_disabled.gz"
-NEOGO_VERSION=0.99.0
+NEOGO_VERSION=0.99.1-pre-180-gcbc693d5
```

Comment:
Is it in public?

Reply:
yes, currently that is the latest public image: https://hub.docker.com/layers/neo-go/nspccdev/neo-go/0.99.1-pre-180-gcbc693d5/images/sha256-f9da801a7a162b6a044017b0efde6636486235e7c7b9e9d6640fcfaf91dff056?context=explore